packages/server/src/api/muster/muster.controller.ts

```diff
@@ -231,30 +233,76 @@ class MusterController {
 
   /**
    * This method returns the normalized rate (0 - 1.0) of compliance on a given
-   * date for a given unit, against  of the unit's muster configs.
+   * date for a given unit, against all of the unit's muster configs.
    */
   async getMusterComplianceByDate(
     req: ApiRequest<{ orgId: number; unitName: string }, null, GetMusterComplianceByDateQuery>,
     res: Response<GetMusterComplianceByDateResponse>,
   ) {
-    const outValue = { musterComplianceRate: 0 };
-    const reportDate = dateFromString(req.query.timestamp);
-    if (!reportDate) {
-      throw new BadRequestError('Missing reportDate.');
+    const outValue = { musterComplianceRate: 0, isoDate: req.query.isoDate };
```

Comment:
This approach below gets the value 1st and then sets the response, this flow seems more logical to me...This might be a matter of a taste. 
```suggestion
      const musterComplianceRate = await getMusterComplianceByDate(req.params.orgId, req.params.unitName, req.query.isoDate);
    res.json({ musterComplianceRate, isoDate: req.query.isoDate });
```